app/containers/markdown/new/Emoji.tsx

```diff
@@ -16,7 +16,7 @@ interface IEmojiProps {
 
 const Emoji = ({ block, isBigEmoji }: IEmojiProps) => {
 	const { theme } = useTheme();
```

Comment:
Since we're working on the file, I think it's worth removing the theme in the places it's being used.